lispusers/DEMO.TEDIT

```diff
@@ -0,0 +1,29 @@
+
+DEMO -- utilities for running demos / tutorials in Medley
+
+includes
+   OPEN-URL (URL)  
+                 (rename of ShellBrowse)
+   MEDLEY-CONTRIB(REPO)
+                  shows GitHub contributors to given repo
+                  uses ShellBrowse
+    BKSYSOBJ(string)
+
+
+DEMO-*.TEDIT
+    contains scripts / TEDIT file talks
+    add your own
+
+
+
+BKSYSOBJ is the start of a facility
+
+(TEDIT.INSERT.OBJ (BKSYSOBJ �(CONS NIL�)     (TEXTSTREAM(WHICHW)]
```

Comment:
Should be `TEDIT.INSERT.OBJECT`?

When evaluating `(TEDIT.INSERT.OBJECT (BKSYSOBJ (CONS NIL) (TEXTSTREAM (WHICHW]` when the pointer is on the TEdit window with `DEMO.TEDIT` open I get the error `BKSYSOBJ is an undefined function` as in this screenshot:

![tedit-demo](https://github.com/Interlisp/medley/assets/10342479/490237e9-384f-47f9-affd-1180d65f7dc1)